### ISSUE TITLE
[XamarinProfiler-Windows]: Accessibility: Color Contrast: MAS1.4.3: "Size text" in snapshot popup window is not having color contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#425]
### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Profiler through ".mlpd" file.
### Steps to Reproduce

1. Follow the Pre-requisite to launch the xamarin profiler.
2. Select the available instruments to "Allocations" and hit "Start Profiling"button.
3. This will invoke "Allocations" profiling page. Navigate to "Snapshot" icon present at top of Allocations progress table and invoke it. Take a screenshot of the same..
4. Verify the color contrast ratio of "Size in MB" (red color) text using the screenshot.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 2.5:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text

### Screenshot
![2 snapshot](https://user-images.githubusercontent.com/22956130/41704380-b3d0596a-7553-11e8-8bda-6983d2775249.png)

### Environment
Microsoft Windows version 1803(OS Build 17134.112)
Xamarin Profiler version 1.6.3.342

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-Windows
Wipro
